lib/app/include/ocall.h

```diff
@@ -12,4 +12,5 @@
 #define ENCLAVE_DEFAULT_KBUFFER_SIZE              0x1000UL
 #define ENCLAVE_DEFAULT_KBUFFER         0xffffffe000000000UL
 
-#define OCALL_SYS_WRITE     3
\ No newline at end of file
+#define OCALL_SYS_WRITE     3
+#define OCALL_USER_DEFINED	4
```

Comment:
+@fengerhu1 Erhu, please check whether this ocall number is consistent with the TVM version.

Reply:
In tvm, we only have a ocall that returns the relay page to the user. However, the relay page is not available in the spmp.
As ocall numbers 1-8  are already used in TVM, maybe a new ocall number 9 is fine.

Reply:
Now I just modify the ocall number to 9, Thx